my-first-react-app/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Hello from './hello';
+
+const users = [
+  { _id: 1, name: 'Leia' },
+  { _id: 2, name: 'Luke' },
+  { _id: 3, name: 'Han' }
+];
+/*class hello extends Component {
```

Comment:
Classes should always start with uppercase letter. 
Unused code should be removed.